languages/python/exercises/concept/bools/.docs/introduction.md

```diff
@@ -0,0 +1,26 @@
+Python represents true and false values with the boolean type. There are only two values: `True` and `False`. These values can be bound to a variable:
+
+```python
+true_variable = True
+false_variable = False
+```
+
+We can evaluate boolean expressions using the `and`, `or`, and `not` operators.
+
+```python
+true_variable = True and True
+false_variable = True and False
+
+true_variable = False or True
+false_variable = False or False
+
+true_variable = not False
+false_variable = not True
+```
+
+When writing a function that returns a boolean value, there isn't a specific naming convention.
+
+```python
+def either_true(a, b):
+    a or b
+```
```

Comment:
While very useful information, does the student need to know this to solve the exercise? If not, this is perfect for the `after.md` document.